src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -277,6 +276,19 @@ internal BoundExpression ConvertPatternExpression(TypeSymbol inputType, CSharpSy
                                                    isVar: isVar, patternTypeWasInSource: true, diagnostics: diagnostics);
             }
 
+            // PROTOTYPE(wildcards): handle wildcard designation here
```

Comment:
This will probably break something downstream once we start testing it. The lowering code and semantic model won't know what to make of the missing symbol. I do expect this code to change.